packages/quantic/force-app/main/default/lwc/quanticResultTemplate/quanticResultTemplate.js

```diff
@@ -0,0 +1,19 @@
+import {LightningElement} from 'lwc';
+
+/**
+ * The `QuanticResultTemplate` component is used to construct result templates using predefined and formatted slots.
```

Comment:
@jpmarceau how's this description? I'll link [slots](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.create_components_slots) and the new doc page I mentioned this morning describing how to use it in more detail can be linked as well once it exists.

Reply:
Yep this looks good 👍 
I'll let you know the url of the doc page as soon as I create it 👍